core/Dockerfile

```diff
@@ -42,7 +42,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # install Kopano Core and refresh ca-certificates
 # hadolint currently does not understand the extended buildkit syntax https://github.com/hadolint/hadolint/issues/347
 # hadolint ignore=SC2215,DL3015
-RUN --mount=type=secret,id=repocred,dst=/etc/apt/apt_auth.conf \
+RUN --mount=type=secret,id=repocred,target=/etc/apt/auth.conf.d/kopano.conf \
```

Comment:
`target` seems interchangeably with `dst`, but potentially easier to read (proper word instead of abbreviation) 👍 

Not sure about the move into auth.conf.d. What value does it give to honor other repo credentials in this repository?

Reply:
As stated in https://manpages.debian.org/testing/apt/apt_auth.conf.5.de.html:

> The APT auth.conf file /etc/apt/auth.conf, and .conf files inside /etc/apt/auth.conf.d can be used to store login information in a netrc-like format with restrictive file permissions.

Reply:
I know what apt_auth.conf can be used for, that quote does not say however why auth.conf.d would be preferred.

Or are you hinting the file should be named `auth.conf` instead of `apt_auth.conf`?

Reply:
@fbartels yeah, either name it auth.conf, or use a custom name and put it to auth.conf.d directory (which I would prefer)

Reply:
ok, then this change makes sense. Must have slipped through since my outgoing ip is whitelisted on that server, but I am pretty sure I tested this on an out of network device as well.